### ISSUE TITLE
How do I use zfsnap to backup snapshots of one pool to another?

### ISSUE THREAD

**Issue by jdrch** [zfsnap/zfsnap#106]
I have a BSD PC with a /home ZFS pool that sits on a single SSD. I'd like to have snapshots of this pool stored on another pool on a separate vdev on the same PC. How do I go about this using `zfsnap` in a `cron` job? The examples given in the documentation assume snapshots on the same block device. I assume it might be something like:

`zfsnap snapshot -r HomePool | OtherPool/BackupLocation/`

This should pipe the output from the zfsnap operation to a location on the other pool. Is this correct, or am I missing something?

Someone suggested I pipe the output of `zfsnap` to `zfs send` then `zfs receive`, but how do I pass that exact snapshot through to `zfs send`, since `zfs send` expects a snapshot as its input and due to how `zfsnap` works I won't know the name of the snapshot a priori?

Sorry if the question sounds stupid but I'm not scripting expert (trying to learn, though.)

In other words what I'm saying is:

`zfsnap snapshot -r srcPool | zfs send srcPool/snap | zfs receive dstPool/location`

probably won't work because `srcPool/snap` is not the location of the snapshot generated by `zfsnap snapshot -r srcPool`. So how do I pass the output of the latter to `zfs send`?

Or is there a different method I should be using?

**EDIT:** How about:

`zfsnap snapshot -r srcPool | zfs send | zfs receive dstPool/location`

Does that look right?

**Comment by asomers** [comment]
You don't.  zfsnap only manages snapshots; it doesn't replicate.  For replication, I suggest [zrepl](https://zrepl.github.io/).

**Comment by jdrch** [comment]
Gotcha, thanks.
